CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### 0.1.0
```

Comment:
I think this should be `0.0.10`. It's still a minor change even though for the case of updating the data the second and third number could be quite similar. We can have 2 digits numbers anyway

Reply:
Many thanks @Levia I wasn't sure about that! I will fix it.